int32 posx
int32 posy
---
bool reached
